tests/scalers/azure/azure_pipelines_aad_wi/azure_pipelines_aad_wi_test.go

```diff
@@ -0,0 +1,317 @@
+//go:build e2e
```

Comment:
I'm afraid about collisions during e2e tests because this test uses the same resources as azure pipelines test. Does it make sense creating another pipeline and agent pool for splitting them?